src/core/PathTracingSceneGenerator.js

```diff
@@ -73,6 +73,8 @@ function getLights( objects ) {
 
 }
 
+let materialUuids = null;
+
```

Comment:
This should be assigned on the `PathTracingSceneGenerator` instance itself. Otherwise if we have two different instances of the generator created they will overwrite each others results.